c++/Calculator.py

```
@@ -0,0 +1,52 @@
+        
+        if choice in ['1', '2', '3', '4']:
+            try:
+                num1 = float(input("Enter first number: "))
+                num2 = float(input("Enter second number: "))
+                
+                if choice == '1':
+                    print(f"{num1} + {num2} = {add(num1, num2)}")
+                elif choice == '2':
+                    print(f"{num1} - {num2} = {subtract(num1, num2)}")
+                elif choice == '3':
+                    print(f"{num1} × {num2} = {multiply(num1, num2)}")
```

Ruff (0.14.1)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)